homeassistant/components/homekit/type_fans.py

```diff
@@ -129,15 +129,18 @@ def set_speed(self, value):
         params = {ATTR_ENTITY_ID: self.entity_id, ATTR_SPEED: speed}
         self.call_service(DOMAIN, SERVICE_SET_SPEED, params, speed)
 
+    def _set_state(self, state):
```

Comment:
Let's rename this to `_set_hk_state` since this is setting it on the HK side while the other methods are changing things on HA side, including `set_state`. Also please add a doc string to indicate what the method does.

Reply:
Okay, thanks for the feedback, I've updated the patch.